static/js/components/email/lib.js

```diff
@@ -55,7 +57,30 @@ export const SEARCH_RESULT_EMAIL_CONFIG: EmailConfig = {
     emailState.inputs.body || '',
     emailState.searchkit.buildQuery().query,
     emailState.inputs.sendAutomaticEmails || false,
-  ])
+  ]),
+
+  renderRecipients: (filters?: Array<any>) => {
+    if (!filters || filters.length <= 0) {
+      return null;
+    }
+    const renderFilterOptions = _.map(filters, filter => (
```

Comment:
It would be good to avoid creating this function at runtime - an easy way would be to use `R.map` instead, then it could be declared, for instance, at the top of this module, like:

```js
const renderFilterOptions = R.map(filter => (
  <div className="sk-selected-filters-option sk-selected-filters__item" key={filter.id}>
    <div className="sk-selected-filters-option__name">
      {filter.name}: {filter.value}
    </div>
  </div>
));
```

then `renderFilterOptions` will be a function that takes an array of filters and returns an array of React components, and we can avoid re-creating that function every time we call `renderRecipients`.